mysql-test/suite.pm

```diff
@@ -72,8 +72,8 @@ sub skip_combinations {
   $skip{'include/have_ssl_communication.inc'} =
   $skip{'include/have_ssl_crypto_functs.inc'} = 'Requires SSL' unless $ssl_lib;
 
-  $skip{'main/openssl_6975.test'} = 'no or wrong openssl version'
-    unless $openssl_ver ge "1.0.1d" and $openssl_ver lt "1.1.1";
```

Comment:
the second condition was added in 74f184aff253 "Fix tests not to fail with OpenSSL 1.1.1 with TLSv1.3". May be your replace fixes that and the test no longer fails with TLSv1.3. If it doesn't ­— sure, thanks for removing the unnecessary limitation.

Reply:
Indeed, this test was only running if MariaDB was built with an old OpenSSL version. This test is now fixed to be generic, and to run on any version of OpenSSL (and we no longer have the situation that new OpenSSL are used untested).